src/main/webapp/index.html

```diff
@@ -26,5 +28,35 @@
           <i class="fas fa-bars"></i>
       </div>
     </header>
+
+    <!-- main page content -->
+    <div id="main-container">
+      <div id="slogan">
+        <div id="slogan-emoji-container">
+          <div id="slogan-container">
+            <h2 id="title-slogan" class="unselectable">find free food, anywhere</h2>
+            <div id="emoji-container">
+              <img src="./assets/bobaemoji.svg" alt="Emoji of boba drink" class="emoji" id="boba-emoji">
+              <img src="./assets/burgeremoji.svg" alt="Emoji of burger" class="emoji" id="burger-emoji">
+              <img src="./assets/pizzaemoji.svg" alt="Emoji of pizza slice" class="emoji" id="pizza-emoji">      
+            </div>  
+          </div>
+          <p class="subtitle" class="unselectable">connecting students and club organizations across college campuses</p>        
+        </div>
+        <div class="selectdiv">
+          <select name="colleges" id="colleges-select">
+            <option value="" disabled selected hidden>pick your college to get started</option>
+          </select>
+        </div>
+      </div>
+      <div id="map-preview">
+        <svg width="200" height="20" id="map-circles">
+          <circle cx="10" cy="10" r="8" stroke="#9D5A5A" stroke-width="1" fill="transparent"></circle>
```

Comment:
Since there's three circle types with the same CSS, would it be better to create a class in CSS for these style changes? Also, nit: google style for CSS asks for lowercase colors!

Reply:
I did some research into this, and it turns out that these aren't CSS attributes, they're 'geometric properties'. It seems that these properties have only been added to the CSS standard very recently, and so not all browsers support styling svgs through CSS (see [here](https://stackoverflow.com/questions/14255631/style-svg-circle-with-css)). As a result, it looks like the most backwards-compatible way to style these circles is through the attributes on the HTML :)

Reply:
You're 100% right about lowercase colors though, will fix!

Reply:
Gotcha, makes sense!